addons/misc/functions/fnc_slingArmband.sqf

```diff
@@ -0,0 +1,239 @@
+#include "script_component.hpp"
+/*
+ * Author: Miss Heda
+ * Attaches a armband to the according limb.
+ *
+ * Arguments:
+ * 0: Player <OBJECT>
+ * 1: String
+ * 2: Number
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, "kat_armband_red_cross", 0] call kat_misc_fnc_slingArmband;
+ *
+ * Public: No
+ */
+
+params ["_unit", "_armbanditem", "_limbnumber"];
+
+private _str = missionNamespace getVariable [QGVAR(armbandSlingLeftArm), ""];
+private _array = [_str, ","" ", true] call FUNC(stringToArray);
+_unit setVariable [QGVAR(armbandSlingLeftArmPos), _array, true];
+
+private _str = missionNamespace getVariable [QGVAR(armbandSlingRightArm), ""];
+private _array = [_str, ","" ", true] call FUNC(stringToArray);
+_unit setVariable [QGVAR(armbandSlingRightArmPos), _array, true];
+
+private _str = missionNamespace getVariable [QGVAR(armbandSlingLeftLeg), ""];
+private _array = [_str, ","" ", true] call FUNC(stringToArray);
+_unit setVariable [QGVAR(armbandSlingLeftLegPos), _array, true];
+
+private _str = missionNamespace getVariable [QGVAR(armbandSlingRightLeg), ""];
+private _array = [_str, ","" ", true] call FUNC(stringToArray);
+_unit setVariable [QGVAR(armbandSlingRightLegPos), _array, true];
+
+private _str = missionNamespace getVariable [QGVAR(armbandSlingLeftArmRotation), ""];
+private _array = [_str, ","" ", true] call FUNC(stringToArray);
+_unit setVariable [QGVAR(armbandSlingLeftArmRot), _array, true];
+
+private _str = missionNamespace getVariable [QGVAR(armbandSlingRightArmRotation), ""];
+private _array = [_str, ","" ", true] call FUNC(stringToArray);
+_unit setVariable [QGVAR(armbandSlingRightArmRot), _array, true];
+
+private _str = missionNamespace getVariable [QGVAR(armbandSlingLeftLegRotation), ""];
+private _array = [_str, ","" ", true] call FUNC(stringToArray);
+_unit setVariable [QGVAR(armbandSlingLeftLegRot), _array, true];
+
+private _str = missionNamespace getVariable [QGVAR(armbandSlingRightLegRotation), ""];
+private _array = [_str, ","" ", true] call FUNC(stringToArray);
+_unit setVariable [QGVAR(armbandSlingRightLegRot), _array, true];
+
+switch (_limbnumber) do 
+{
+    // Attach to left arm
+    case 0: { 
+        switch (_armbanditem) do 
+        {
+            case "kat_armband_red_cross": { 
+                _armbandLA = "Kat_armbandRC" createVehicle position _unit;
+                _armbandLA attachTo [_unit, _unit getVariable [QGVAR(armbandSlingLeftArmPos), [0,0,0]],"leftforearm",true];
```

Comment:
spaces after ','  (I will only mention it once so i dont spam the whole request)